src/BinSkim.Rules/PERules/BA2006.BuildWithSecureTools.cs

```diff
@@ -54,6 +56,15 @@
 
         private const string MIN_XBOX_COMPILER_VER = "MinimumXboxCompilerVersion";
 
+        public static PerLanguageOption<Version> MinimumCCompilerVersion { get; } =
+            new PerLanguageOption<Version>(AnalyzerName, nameof(Language.C), defaultValue: () => new Version());
+        public static PerLanguageOption<Version> MinimumCxxCompilerVersion { get; } =
+            new PerLanguageOption<Version>(AnalyzerName, nameof(Language.Cxx), defaultValue: () => new Version());
+        public static PerLanguageOption<Version> MinimumUnknownCompilerVersion { get; } =
+            new PerLanguageOption<Version>(AnalyzerName, nameof(Language.Unknown), defaultValue: () => new Version());
+        public static PerLanguageOption<Version> MinimumXboxCompilerVersion { get; } =
+            new PerLanguageOption<Version>(AnalyzerName, MIN_XBOX_COMPILER_VER, defaultValue: () => new Version());
```

Comment:
>MIN_XBOX_COMPILER_VER [](http://example.com/codeflow?start=57&length=21)

please get rid of this constant and use nameof(PropertyName) as we do elsewhere.